src/pages/Location/Location.js

```diff
@@ -65,14 +65,14 @@ const LocationComponent = () => {
   const councilLabelRedux = useSelector((stateRedux) => stateRedux.mapData.councilLabel);
   const progressRedux = useSelector((stateRedux) => stateRedux.sharedData.progress);
   const myCoverCropListLocationRedux = useSelector((stateRedux) => stateRedux.sharedData.myCoverCropListLocation);
-  const regionShorthand = useSelector((stateRedux) => stateRedux.mapData.regionShorthand);
   const accessTokenRedux = useSelector((stateRedux) => stateRedux.userData.accessToken);
   const userFieldRedux = useSelector((stateRedux) => stateRedux.userData.field);
 
   // useState vars
+  const [selectedRegion, setRelectedRegion] = useState();
   const [handleConfirm, setHandleConfirm] = useState(false);
-  const [selectedZone, setselectedZone] = useState();
   const [locZipCode, setLocZipCode] = useState();
+  const [regionShorthand, setRegionShorthand] = useState();
```

Comment:
this should be in redux, it's also used in landing.js

Reply:
I don't see this in landing but I assume you mean it will be getting used at some point so I made the updates to put regionShorthand back into redux and remove it from state in location.js